metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -457,6 +571,9 @@ class LookerExplore:
     upstream_views: Optional[
         List[ProjectInclude]
     ] = None  # captures the view name(s) this explore is derived from
+    upstream_views_file_path: Dict[str, Optional[str]] = dataclasses_field(
```

Comment:
why is this `Optional[str]`? it looks like it should be `str` since every view has a file path

Reply:
Please go through function `create_upstream_views_file_path_map` and `get_view_file_path` . The function is looking for view-name on `lkml_fields` if view-name is found then return view-file-path othrewise return None